datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -56,6 +65,28 @@ pub(crate) enum ExecutionState {
 use super::order::GroupOrdering;
 use super::AggregateExec;
 
+/// This encapsulates the spilling state
+struct SpillState {
+    /// If data has previously been spilled, the locations of the
+    /// spill files (in Arrow IPC format)
+    spills: Vec<NamedTempFile>,
```

Comment:
I will change from `NamedTempFile` to `RefCountedTempFile` once https://github.com/apache/arrow-datafusion/pull/7538/files is merged

Reply:
Updated